src/main/webapp/app/shared/sidebar/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -2,14 +2,26 @@
     <div
         id="test-accordion-item-container-{{ i }}"
         class="pointer"
-        [ngClass]="{ 'd-none': !(groupedData[groupKey].entityData | searchFilter: ['title', 'type'] : searchValue)?.length }"
+        [ngClass]="{ 'd-none': sidebarType !== 'conversation' && !(groupedData[groupKey].entityData | searchFilter: ['title', 'type'] : searchValue)?.length }"
     >
         <div id="test-accordion-item-header" class="control-label d-flex align-items-center justify-content-between py-2 bg-module" (click)="toggleGroupCategoryCollapse(groupKey)">
             <div class="my-2">
-                {{ 'artemisApp.courseOverview.sidebar.' + groupKey | artemisTranslate | titlecase }}
+                @if (sidebarType !== 'conversation') {
```

Comment:
Introduce a more generic flat like `showLeadingIcon` you can set its default to `false` and for the conversation data to `true`.